doc/proposals/helm-operator.md

```diff
@@ -0,0 +1,127 @@
+## Helm Operator Proposal
+
+### Background
+
+As was mentioned in the [Ansible Operator Proposal](./ansible-operator.md), not everyone is a golang developer, so the SDK needs to support other types of operators to gain adoption across a wider community of users.  
+
+[Helm](https://helm.sh/) is one of the most widely-used tools for Kubernetes application management, and it bills itself as the "package manager for Kubernetes." Operators serve a nearly identical function, but they improve on Helm's concepts by incorporating an always-on reconciliation loop rather than relying on an imperative user-driven command line tool. By integrating Helm's templating engine and release management into an operator, the SDK will further increase the number of potential users by adding the ability to deploy Charts (e.g. from Helm's [large catalog of existing Charts](https://github.com/helm/charts)) as operators with very little extra effort.
+
+### Goals
+
+The goal of the Helm Operator will be to create a fully functional framework for Helm Chart developers to create operators. It will also expose a library for golang users to use Helm in their operator if they so choose. These two goals in conjunction will allow users to select the best technology for their project or skillset.
+
+### New Operator Type
+
+This proposal creates a new type of operator called `helm`. The new type is used to tell the tooling to act on that type of operator.
+
+### Package Structure
+
+Packages will be added to the operator-sdk. These packages are designed to be usable by the end user if they choose to and should have a well documented public API. The proposed packages are:
+
+* /operator-sdk/pkg/helm/client
+  * Will contain a helper function to create a Helm client from `controller-runtime` manager.
+
+* /operator-sdk/pkg/helm/controller
+  * Will contain the Helm controller.
```

Comment:
Is the controller going to be exported? Or just an ‘Add’ method?

Reply:
Just an `Add` method that uses the exported `HelmOperatorReconciler`.